seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java

```diff
@@ -337,6 +337,9 @@ default Object[] sampleDataFromColumn(
                     if (count % samplingRate == 0) {
                         results.add(rs.getObject(1));
                     }
+                    if (Thread.currentThread().isInterrupted()) {
```

Comment:
Need to change this?

https://github.com/apache/seatunnel/blob/329b849fbe4ff6aad4f9c6ce2438b903554a54fa/seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/splitter/JdbcSourceChunkSplitter.java#L101

Reply:
done, added